.github/workflows/build.yaml

```diff
@@ -38,4 +38,7 @@ jobs:
       run: ./build.sh package
     - name: Publish to PyPI Linux
       if: runner.os == 'Linux'
```

Comment:
@PeterVenable The branch check on line 31 is also important; otherwise PR builds will try to publish the package.